README.md

```diff
@@ -300,6 +300,36 @@ const MyAwesomeForm = () => {
 };
 ```
 
+## Validation helper
+
+In case you want to use several validation functions, there is `combineValidators` helper which can help you to set several validation functions for a field. This helper works with sync and async functions (by using promise for async).
+
+```tsx
+import { useForm, combineValidators } from "react-ux-form";
+
+const validateRequired = (value: string) => {
+  if (!value) {
+    return "required";
+  }
+};
+
+const validateEmail = (email: string) => {
+  if (!/.+@.+\..{2,}/.test(email)) {
+    return "invalid email";
+  }
+};
+
+const MyAwesomeForm = () => {
+  const { Field, submitForm } = useForm({
+    email: {
+      initialValue: "",
+      // will run each validation function until an error is found
+      validate: combineValidators(validateRequired, validateEmail),
```

Comment:
Could be nice to add a third validator with a condition (`false`) to show it's possible.

Reply:
ok will add this